hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OzoneAclUtil.java

```diff
@@ -180,6 +180,25 @@ public static boolean inheritDefaultAcls(List<OzoneAcl> acls,
     return false;
   }
 
+  /**
+   * Helper function to convert the scope of ACLs to DEFAULT.
+   * This method is called in ACL inheritance scenarios.
+   * @param acls
+   */
+  public static void toDefaultScope(List<OzoneAcl> acls) {
+    acls.forEach(a -> a.setAclScope(DEFAULT));
+  }
+
+  /**
+   * Helper function to filter the DEFAULT scope ACLs.
+   * This method is called in ACL inheritance scenarios.
+   * @param acls
+   */
+  public static List<OzoneAcl> filterDefaultScope(List<OzoneAcl> acls) {
```

Comment:
This is not needed as can use inheritDefaultAcl can be used.

Reply:
> This is not needed as can use inheritDefaultAcl can be used.

Used method `getAclsForDir` similar `getAclsForKey`.